### DISCUSSION TITLE
Uma sugestão para a seção de "Magias"

### DISCUSSION THREAD

**Discussion by V3rity11** [alyssapiresfernandescefet/openrpg#11] [Ideas]
Primeiro, parabéns pelo seu esforço com este projeto.

Segundo, uma boa ideia seria tornar o dano das magias opcional, ou talvez ocultá-lo quando deixado em branco, tendo em vista que nem todos os tipos de magia são necessariamente de ataque.

**Comment by lucaspiresfernandes** [comment]
Olá, @V3rity11! Muito obrigada!

Então, existe essa funcionalidade! O problema é que a documentação aqui no GitHub tá atrasada e não mostra esse caso, mas o sistema trata o valor "-" como não-aplicável. Apesar de não permitir campos em branco, você pode deixar um "-" para dizer ao sistema que aquele valor é irrelevante.

Quando esse valor está no dano de uma magia, verá que o dado ao lado na ficha do jogador irá desaparecer.

**Comment by lucaspiresfernandes** [comment]
Não se preocupe com a documentação; estou preparando uma gama de ferramentas que estou ansiosa para mostrar em público!
